src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const rows = sourceString.split(';').filter(Boolean);
```

Comment:
You can chain the 'split' and 'filter' methods for better readability. Also, the 'filter(Boolean)' is a clever way to remove empty strings, but make sure it's clear to other developers who might read your code.